cost/azure/hybrid_use_benefit/azure_hybrid_use_benefit.pt

```diff
@@ -132,14 +132,23 @@ datasource "azure_instances" do
     end
 end
 
+datasource "datasource_ds_azure_pricing" do
```

Comment:
Datasource names should begin with ds_, not datasource_ds_. See:

https://flexera.atlassian.net/wiki/spaces/FP/pages/2184642622/Policy+Best+Practices#Block-Naming-Conventions